MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionViewQtGUI.h

```diff
@@ -212,9 +212,7 @@ class MANTIDQT_CUSTOMINTERFACES_DLL EnggDiffractionViewQtGUI
                            const std::string &spectrum,
                            const std::string &type) override;
 
-  void plotVanCurvesCalibOutput() override;
-
-  void plotDifcZeroCalibOutput(const std::string &pyCode) override;
+  void plotCalibOutput(const std::string &pyCode) override;
```

Comment:
Note that along these lines we could go further and just have one method to run any plotting/non-plotting Python code in the view. The only reason to have this in the view is that the PythonRunner is part of the view (Mantid Qt sub window), but other than that all the preparation of the code can be done in the presenter.
But that would be for next rounds of refactoring.